### PR TITLE
[INF-0002] fix comment about resource not fully bound.

### PR DESCRIPTION
**Pull request by python3kgae**
Fix the comment in resource not fully bound example.

### CODE REVIEW

**Comment by damyanp** [comment]
Can you update the PR title to something more descriptive please. "fix missing review issue." won't make any sense to someone looking through the commits in the future.

File: proposals/infra/INF-0002-root-signature-in-clang.md [hunk 1]
```diff
@@ -467,7 +467,7 @@ elimation has completed.
 
 Most values like ShaderVisibility/ParameterType are covered by syntactical 
 checks in Sema.
-Only list special rule here.
+The additional sematic rules not already covered by the grammar are listed here.
```
Comment (damyanp) [inline proposals/infra/INF-0002-root-signature-in-clang.md:470]:
```suggestion
The additional semantic rules not already covered by the grammar are listed here.
```

File: proposals/infra/INF-0002-root-signature-in-clang.md [hunk 2]
```diff
@@ -626,7 +626,7 @@ TODO
 
 #### Resource used in DXIL must be fully bound in root signature.
 ```
-  // B is bound to t1, but not in root descriptor.
+  // B is bound to t1, but not in root signature.
```
Comment (damyanp) [inline proposals/infra/INF-0002-root-signature-in-clang.md:629]:
```suggestion
  // B is bound to t1, but no root parameters cover t1.
```

Comment (damyanp) [reply proposals/infra/INF-0002-root-signature-in-clang.md:629]: The root signature says nothing about resources, it tells you about registers.

**Review by dmpots — APPROVED** [review]

**Comment by python3kgae** [comment]
> Can you update the PR title to something more descriptive please. "fix missing review issue." won't make any sense to someone looking through the commits in the future.

Updated.

**Review by damyanp — APPROVED** [review]

**Pull request merged**